apps/pie-docs/src/_includes/nav.njk

```diff
@@ -50,16 +66,17 @@
             {% for category in section %}
                 {% if category.url %}
                     <li>
-                        <details {% if page.url and page.url.includes(category.url) %} open{% endif %}>
-                            {{ summary(category.title, category.key) }}
+                        <details>
+                            {# hasIndexPage is an attribute that is set in eleventyNavigation metadata when a category has an index page #}
+                            {{ summary(category.title, category.url, category.children[0].url, category.hasIndexPage, category.key) }}
                             {% if category.children.length %}
                             {# build up a list of subCategories for each subpages (used in component pages) #}
                                 <ul class="c-nav-list">
                                     {% for subpage in category.children %}
                                         {% if subpage.url %}
                                             {% if subpage.children.length and subpage.subPageDropdown %}
-                                                <details {% if page.url and page.url.includes(subpage.key) %} open{% endif %}>
-                                                    {{ summary(subpage.title) }}
+                                                <details>
```

Comment:
same as above

Reply:
Added this line back and removed animation when clicking on links as agreed with the team :)